libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -150,6 +150,25 @@ static const char * pNoLowLevelMbedTlsCodeStr = "<No-Low-Level-Code>";
  */
 #define pkcs11_MAX_EC_PRIVATE_KEY_DER_SIZE    pkcs11_PRIVATE_EC_PRIME_256_DER_SIZE
 
+/**
+ * @ingroup pkcs11_macros
+ * @brief Length of bytes to contain an EC point.
+ *
+ * This port currently only uses prime256v1, in which the fields are 32 bytes in
+ * length. The public EC point is as long as the curve's fields * 2 + 1.
+ * so the EC point for this port is (32 * 2) + 1 bytes in length.
+ *
+ * mbed TLS encodes the length of the point in the first byte of the buffer it
+ * receives, so an additional 1 byte in length is added to account for this.
+ *
+ * In addition to this, an additional 1 byte is added to store information
+ * indicating that the point is uncompressed.
+ *
+ * @note This length needs to be updated if using a different curve.
+ *
+ */
+#define pkcs11EC_POINT_LENGTH                 67
```

Comment:
This would be more readable.  The compiler will compute this and there will be no runtime penalty:

```c
// 32 points of 2 bytes each + 1 point length byte, 1 length byte, 1 type (uncompressed) byte
 #define pkcs11EC_POINT_LENGTH                 ( ( 32 ) * ( 2 ) + 1 + 1 + 1 )
```

Reply:
Done